sonar-duplications/src/main/java/net/sourceforge/pmd/cpd/SourceCode.java

```diff
@@ -59,9 +62,7 @@
     protected abstract Reader getReader() throws Exception;
 
     protected List<String> load() {
-      LineNumberReader lnr = null;
-      try {
-        lnr = new LineNumberReader(getReader());
+      try (LineNumberReader lnr = new LineNumberReader(getReader())) {
```

Comment:
what? Jacoco does not support try-with-resource?
That would be the reason I haven't many of them in code yet.